.github/ISSUE_TEMPLATE/launch.md

```diff
@@ -0,0 +1,158 @@
+- [ ] Verify analytics tools are only reporting from the production environment
+      or reporting separately depending on requirements.
+
+## QA Site
+- [ ] Determine if a QA site is needed.
+- [ ] Confirm that `robots.txt` or the corresponding HTTP header is configured
+      to block traffic to the QA site.
+- [ ] Implement password protection with HTTP Authorization if required.
+- [ ] Verify that proper testing API keys are configured if needed.
+
+## Performance
+- [ ] Enable Drupal Page caching.
+- [ ] Enable Drupal Big Pipe caching if needed.
+- [ ] Verify that custom render array cache metadata is set and tested.
+- [ ] Determine if CSS/JavaScript aggregation should be enabled.
+- [ ] Verify that development related services are disabled (e.g.
+      `development.services.yml`)
+- [ ] Verify Redis connectivity and Redis is used for caching.
+- [ ] Verify that Redis is using the correct cache prefix if applicable.
+- [ ] Configure and test the CDN if applicable.
+- [ ] Verify the CDN is serving HITs if applicable.
+- [ ] Verify Varnish is serving HITs if applicable.
+- [ ] Configure and test the load balancer if applicable.
+- [ ] Verify that cron runs from pages requests are disabled.
+
+## Development
+- [ ] Verify that the mapping in `sites.php` supports the production domain.
+- [ ] Verify automatic deployments are working.
+- [ ] Verify that PHP error messages are suppressed.
+- [ ] Disable database logging.
+- [ ] Enable syslog.
+- [ ] Verify that config splits are enabled/disabled appropriately in all
+      environments.
+- [ ] Check logs for recurring PHP notices/warnings/errors.
+- [ ] Fix recurring issues found in the site logs.
+- [ ] Check the browser console for recurring errors.
+- [ ] Fix recurring issues found in the browser console.
+- [ ] Verify that Stage File Proxy is disabled on PROD.
+- [ ] Enable and configure Stage File Proxy for non-production environments.
+- [ ] Verify that config exports and imports cleanly.
+- [ ] Determine the necessary cron frequency.
+- [ ] Configure external cron trigger (Jenkins/Crontab/etc).
+- [ ] Verify that the deployment process works.
+- [ ] Check site error logs and fix persistent issues.
+- [ ] Verify that Drupal's status report page is clear of issues.
+- [ ] Verify that migrations are complete.
+- [ ] Determine if git branching strategy or merge rules should be adjusted post
+      launch.
+- [ ] Verify that no API keys or sensitive credentials are stored in code.
+- [ ] Verify if "helper" node types and/or taxonomy term pages should appear and
+      hide or block access as needed.
+- [ ] Verify that alias patterns and updates/redirects are working correctly.
+- [ ] Verify that `$settings['hash_salt']` is set.
+- [ ] Verify that `$settings['update_free_access'] = FALSE;` is set.
+- [ ] Verify that `$settings['omit_vary_cookie']` is configured correctly.
+- [ ] Verify that `$settings['file_public_path']` and `file_private_path` are
+      configured correctly.
```

Comment:
Perhaps we should include a twig method whitelist from the get-go?

```php
$settings['twig_sandbox_whitelisted_methods'] = [
  'id',
  'label',
  'bundle',
  'get',
  '__toString',
  'toString',
  'referencedEntities',
];
```

Reply:
This is more functionality development related vs launch development.